banyand/measure/measure_topn.go

```diff
@@ -190,8 +190,8 @@ func (t *topNStreamingProcessor) writeData(eventTime time.Time, timeBucket strin
 						Tags: append([]*modelv1.TagValue{
 							// MeasureID
 							{
-								Value: &modelv1.TagValue_Id{
```

Comment:
The `measure_id`'s `indexRule` is removed. @lujiajing1126 Are there any potential side effects to consider?